.gitignore

```diff
@@ -15,7 +15,6 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
```

Comment:
Required because we have a module called `lib` inside our Pylons test folder. The name is hard-coded in Pylons.